server/index.js

```diff
@@ -0,0 +1,27 @@
+'use strict';
+const
+    express = require('express'),
+    bodyParser = require('body-parser');
+let server = express(),
```

Comment:
You can probably convert these to `const` as well since you are never reassigning these variables